tests/core/interfaces-gpio-chardev/task.yaml

```diff
@@ -9,6 +9,10 @@ details: |
 # snap is not built for arm.
 systems: [-ubuntu-core-18-*, -ubuntu-core-20-*, -ubuntu-core-22-*, -ubuntu-core-*-arm-*]
 
+# This is explicitly disabled until gpio-aggregator configfs support
+# lands in kernel and snapd.
+manual: true
```

Comment:
Would it make sense to enable the test anyway and make sure that an attempt to enable the interface results in an error? This will make it fail again once relevant kernel gets updated so we'll know when to update the test once more.